its/pom.xml

```diff
@@ -18,7 +18,7 @@
   <inceptionYear>2013</inceptionYear>
 
   <properties>
-    <sonar-ws.version>8.9.0.43852</sonar-ws.version>
+    <sonar-ws.version>10.0.0.65447</sonar-ws.version>
```

Comment:
I'm afraid that as this artifact is not a release, it will be deleted in a few weeks. I guess the 8.9.0.43852 corresponds to the latest 8.9 LTS... You can probably use the build number of the 9.9 LTS here :)

Reply:
As discussed, using 9.9 won't solve the problem in itself. Though I'll do this, as it makes more sense, and fix the actual problem with another method (ie. doing the API call, not rely on sonar-ws)